fleetctl/start.go

```diff
@@ -87,5 +87,10 @@ func runStartUnit(cCmd *cobra.Command, args []string) (exit int) {
 		return exitVal
 	}
 
+	exitVal = tryWaitForSystemdActiveState(starting)
```

Comment:
Geez, you're right. I'll pass `getBlockAttempts(cCmd)` to that function to check the flags, just like `tryWaitForUnitStates()`.